packages/client/src/open-feature.ts

```diff
@@ -17,6 +17,11 @@ const GLOBAL_OPENFEATURE_API_KEY = Symbol.for('@openfeature/web-sdk/api');
 type OpenFeatureGlobal = {
   [GLOBAL_OPENFEATURE_API_KEY]?: OpenFeatureAPI;
 };
+type NameProviderTuple = {
```

Comment:
We should come up with a better name for this.

Reply:
I have changed "namespace" to "name" and in that case I think this is a good name for this type.

Reply:
renamed to -Record